vspace/src/main/java/edu/asu/diging/vspace/web/staff/EditSpaceLinks.java

```diff
@@ -0,0 +1,43 @@
+package edu.asu.diging.vspace.web.staff;
+
+import java.io.IOException;
+
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+
+import com.fasterxml.jackson.databind.ObjectMapper;
+import com.fasterxml.jackson.databind.node.ObjectNode;
+
+import edu.asu.diging.vspace.core.model.ISpace;
+import edu.asu.diging.vspace.core.services.ISpaceManager;
+
+public class EditSpaceLinks {
```

Comment:
bad class name, it's the superclass for controllers, right? so it should be called controller.